release/apparmor_test.go

```diff
@@ -44,35 +44,6 @@ func (*apparmorSuite) TestAppArmorLevelTypeStringer(c *C) {
 	c.Check(release.AppArmorLevelType(42).String(), Equals, "AppArmorLevelType:42")
 }
 
-func (*apparmorSuite) TestAppArmorLevelTriggersAssesment(c *C) {
-	// Pretend that we know the apparmor kernel and parser features.
-	restore := release.MockAppArmorFeatures([]string{"feature"}, nil, []string{"feature"}, nil)
-	defer restore()
-	// Pretend that we don't know what the state of apparmor is.
-	release.ResetAppArmorAssesment()
```

Comment:
this reset function was used only in these test afaict, should it go away now?

Reply:
Dropped now.